build.sbt

```diff
@@ -59,8 +64,7 @@ lazy val dockerCommon = Seq(
   Docker / daemonUserUid := None,
   dockerPermissionStrategy := DockerPermissionStrategy.CopyChown,
   dockerRepository := Some("snowplow"),
-  scriptClasspath += "/config",
-  Universal / javaOptions ++= Seq("-Dnashorn.args=--language=es6")
+  Universal / mappings += file("LICENSE.md") -> "LICENSE.md"
```

Comment:
Did you deliberately remove `-Dnashorn.args=--language=es6` from this mapping?

Reply:
It is only applicable to the non-distroless (distrofull?) build, because the distroless build overrides the Docker entrypoint. You can see that distroless adds it back in the form of `"-Dnashorn.args=--language=es6"`, and that I moved this line to the non-distroless settings.

Reply:
Same for classpath settings — those are set differently in distroless

Reply:
Oh ignore.  I see now why you moved it.  I like it better now.